addons/web/static/src/views/fields/datetime/datetime_field.js

```diff
@@ -331,7 +338,18 @@ export const dateTimeField = {
                 `Control the number of minutes in the time selection. E.g. set it to 15 to work in quarters.`
             ),
         },
+        {
+            label: _t("Show seconds"),
```

Comment:
don't you have the same problem as in https://github.com/odoo/enterprise/pull/67258/commits